wgsl/index.bs

```diff
@@ -3676,12 +3668,8 @@ module.
 function_decl
   : decoration_list* function_header body_statement
 
-function_type_decl
-  : type_decl
-  | VOID
-
 function_header
-  : FN IDENT PAREN_LEFT param_list PAREN_RIGHT ARROW function_type_decl
+  : FN IDENT PAREN_LEFT param_list (PAREN_RIGHT ARROW type_decl)?
```

Comment:
The `(` comes too early. I think we require the `PAREN_RIGHT` in all cases, move the `(` to before `ARROW)

Reply:
oh snap, nicely spotted!